examples/Support-pipeline/main.bal

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+import ballerina/oauth2;
+import ballerinax/hubspot.crm.pipelines as hspipelines;
+
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string refreshToken = ?;
+configurable string pipelineLabel = "Support Pipeline";
+configurable string objectType = "tickets";
+
+public function main() returns error? {
+    hspipelines:OAuth2RefreshTokenGrantConfig auth = {
+        clientId: clientId,
+        clientSecret: clientSecret,
+        refreshToken: refreshToken,
+        credentialBearer: oauth2:POST_BODY_BEARER
+    };
+    hspipelines:Client hubSpotPipelines = check new ({auth});
```

Comment:
Let's move the pipeline creation outside of the main function. Then you don't have to pass it to each function. We can make the client `final` as well.

```ballerina
final hspipelines:Client hubspotPipelines = check new ({auth});
```